src/constants/common.js

```diff
@@ -131,4 +131,8 @@ module.exports = {
 	},
 	PUSH: 'PUSH',
 	POP: 'POP',
+	AUTH_METHOD: {
+		JWT_ONLY: 'jwt_only',
```

Comment:
@nevil-mathew change to JWT: true,

Reply:
@rakeshSgr The reason for not going with Boolean was to leave scope for future upgrades, like if we need to add more auth methods.
And the change to "JWT: JWT" wouldn't be fully correct because, even in the second method we need JWT, because the mentoring service won't work without the decoded token.